canonical-web-and-design/juju.is
# Linkify site

## Done
Update the site content to match the latest changes and added links from the homepage.

## QA
- Go to the demo and check the homepage matches [the copy doc](https://docs.google.com/document/d/1vn6it-3px_KbplnajPlWPonU6W_xp4JdhE02kNQouZA/edit).
- Ensure all links work

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1427
